AgilefantTimes/Program.cs

```diff
@@ -29,15 +31,17 @@ public static void Main(string[] args)
                     where agilefantSprint.Name.Contains(config.SprintNumber.ToString())
                     select agilefantSprint.Id).First();
 
-                Console.WriteLine("{\"Hours\":[");
+                Console.WriteLine("{\n    \"Hours\":[");
```

Comment:
Haha it could indeed. The pretty printer * should* work for well formed JSON. But I haven't really tested